library/src/main/java/com/getstream/sdk/chat/view/MessageListViewStyle.java

```diff
@@ -142,9 +150,17 @@ public MessageListViewStyle(Context c, AttributeSet attrs) {
 
         readStateTextSize = a.getDimensionPixelSize(R.styleable.MessageListView_streamReadStateTextSize, getDimension(R.dimen.stream_read_state_text_size));
         readStateTextColor = a.getColor(R.styleable.MessageListView_streamReadStateTextColor, Color.BLACK);
-        readStateTextStyle = a.getColor(R.styleable.MessageListView_streamReadStateTextStyle, Typeface.BOLD);
+        readStateTextStyle = a.getInt(R.styleable.MessageListView_streamReadStateTextStyle, Typeface.BOLD);
         // Thread
         threadEnabled = a.getBoolean(R.styleable.MessageListView_streamThreadEnabled, true);
+        // Date Separator
+        dateSeparatorDateTextSize = a.getDimensionPixelSize(R.styleable.MessageListView_streamDateSeparatorDateTextSize, getDimension(R.dimen.stream_date_separator_text));
+        dateSeparatorDateTextColor = a.getColor(R.styleable.MessageListView_streamDateSeparatorDateTextColor, getColor(R.color.stream_gray_dark));
+        dateSeparatorDateTextStyle = a.getInt(R.styleable.MessageListView_streamDateSeparatorDateTextStyle, Typeface.BOLD);
+
+        dateSeparatorLineWidth = a.getDimensionPixelSize(R.styleable.MessageListView_streamDateSeparatorLineWidth, getDimension(R.dimen.stream_date_separator_text));
```

Comment:
Dimension is wrong: `getDimension(R.dimen.stream_date_separator_text)`. Should be probably `stream_date_separator_line_width`